agent/config/parse.go

```diff
@@ -252,6 +252,14 @@ func parseContainerInstancePropagateTagsFrom() ContainerInstancePropagateTagsFro
 	}
 }
 
+func parseNvidiaRuntime() string {
```

Comment:
You shouldn't need this, but instead can set the default value in [here](https://github.com/aws/amazon-ecs-agent/blob/608808bb1a7eceed7e3bb390324c7894a75d2141/agent/config/config_unix.go#L44).

Reply:
ah, missed this one. will modify.